### PR TITLE
PR for Enhancements

### PR DESCRIPTION
**Pull request by SK02K1**
# Enhancements

- favicon added.
- useDocumentTitle custom hook to change the document title after mounting.
- 404 page.
- Toast messages on cart and auth services.
- Scroll to top when a page is mounted.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,19 +1,22 @@
+import { Toaster } from 'react-hot-toast';
 import { Route, Routes } from 'react-router-dom';
 import { Navbar, PrivateRoute, Wrapper } from './components';
 import {
```
Comment (aman11s) [inline src/App.jsx:4]: liked you are using barrel export here

File: src/hooks/useDocumentTitle.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { useState, useEffect } from 'react';
+
+export const useDocumentTitle = (title) => {
```
Comment (aman11s) [inline src/hooks/useDocumentTitle.js:3]: liked you are using custom hooks for changing document title

**Review by aman11s — APPROVED** [review]
Good to merge.

**Review by mohammadkhizerkhan — APPROVED** [review]
clean code every thing looks fine to me, good to merge

**Review by manismk — APPROVED** [review]
Looks Good. Code is easy to understand.
Liked the document title hook.

**Pull request merged**